doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst

```diff
@@ -65,14 +65,16 @@ Procedure
 
    Thanks for checking out the CORTX system; we hope you liked it. Hopefully you'll stick around and participate in our community and help make it even better.
 
-**********
+**********************************
 Validate CSM endpoint via REST API
-**********
-#. Validate the CSM endpoint using below command. Use hostname or ip of your OVA VM.
+**********************************
+
+#. Validate the CSM endpoint using below command. Use hostname or ipaddress as per your VM
    
    ::
    
-        curl --request POST '<https://$HOSTNAME OR https://$IPADDRESS>/api/v2/login?debug' --header 'Content-Tycation/json' -d '{"username":"cortxadmin","password":"Cortxadmin@123"}' -k
+      curl --request POST '<https://$IPADDRESS>:31169/api/v2/login?debug>' --header 'Content-Tycation/json' -d '{"username":"cortxadmin","password":"Cortxadmin@123"}' -k      
```

Comment:
Can we use - `https://<$IPADDRESS>:31169/api/v2/login?debug` . As only IP Address will change based on setup. 
Also, we may dynamically populate it as below 
```
[root@test ~]# echo $(ifconfig eth1 | grep inet -w | awk '{print $2}')
192.168.49.191
```

Reply:
Done, changed accordingly